pywhat/__main__.py

```diff
@@ -4,21 +4,16 @@
 pywhat: Identify Anything.
 """
 
+import platform
 import sys
 
 if __name__ == "__main__":
     major = sys.version_info[0]
     minor = sys.version_info[1]

```

Comment:
Instead of using `sys` here, we can use `python_version`?

> platform.python_version()
> Returns the Python version as string 'major.minor.patchlevel'.
> 
> Note that unlike the Python sys.version, the returned value will always include the patchlevel (it defaults to 0).

So to get major and minor, we can do:

```py
python_version = platform.python_version()
versions = python_version.split(".")
major = versions[0]
minor = versions[1]
```

This saves us from importing `sys` :-)

Reply:
Going to change that!

Reply:
@bee-san, you forgot that sys.argv and sys.exit() are being used. Therefore, I think that these changes are not necessary.